.travis.yml

```diff
@@ -24,6 +24,10 @@ matrix:
   - stage: build
     env: CI_SCRIPT=ci/test-mill-0.sh
     jdk: oraclejdk8
+  - stage: build
+    env: CI_SCRIPT=ci/test-mill-0.5.sh
```

Comment:
Could you rename that script file to something more expressive, e.g. `test-mill-postgresql.sh`. It would it make easier to grasp which script does what.